specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/examples/VpnConnectionPut.json

```diff
@@ -0,0 +1,62 @@
+{
+  "parameters": {
+    "connectionName": "vpnConnection1",
+    "gatewayName": "gateway1",
+    "resourceGroupName": "rg1",
+    "api-version": "2018-06-01",
+    "subscriptionId": "subid",
+    "VpnConnectionParameters": {
```

Comment:
Should be "VpnConnectionParameters" because of https://github.com/Azure/azure-rest-api-specs/pull/3204/files#diff-87bfafdce3f0047871a27733d64bf461R1430

Reply:
Here: "vpnConnectionParameters"